Zotero/Models/Defaults.swift

```diff
@@ -6,6 +6,7 @@
 //  Copyright © 2019 Corporation for Digital Scholarship. All rights reserved.
 //
 
+import AVFAudio
```

Comment:
Doesn't seem to be needed here.
```suggestion
import AVFAudio
```

Reply:
9c74e684f65d01c615c7e788de8685ba1d9edd3c